spec/models/provider_spec.rb

```diff
@@ -271,6 +271,22 @@
     end
   end
 
+  describe "accredited_providers" do
+    let(:provider) { create :provider, accrediting_provider: 'N' }
+
+    # two courses with same accrediting provider to ensure returned data is not duplicated
+    let!(:course1) { create :course, :with_accrediting_provider, provider: provider }
```

Comment:
prefer to be more explicit with `accrediting_provider`

``` ruby
let(:provider) { create :provider, accrediting_provider: 'N' }
let(:accrediting_provider:r) { create :provider }

    let!(:course1) { create :course, accrediting_provider: accrediting_provider, provider: provider }
    let!(:course2) { create :course, accrediting_provider: accrediting_provider, provider: provider }

```

Not sure is the `!` btw